pkg/monitoring/updates/apt.go

```diff
@@ -0,0 +1,57 @@
+package updates
+
+import (
+	"context"
+	"fmt"
+	"os/exec"
+	"strings"
+	"time"
+
+	"github.com/pkg/errors"
+)
+
+type pkgMgrApt struct {
+}
+
+func (a *pkgMgrApt) GetBinaryPath() string {
+	return "/usr/bin/apt-get"
```

Comment:
does `apt-get` always available on new systems with just `apt`?

Reply:
apt-get still ships with the all latest Debian-based systems.

Also, `man apt` on my Ubuntu 18 shows that
> apt provides a high-level commandline interface for the package management system. It is
       intended as an end user interface and enables some options better suited for interactive
       usage by default compared to more specialized APT tools like apt-get(8) and apt-cache(8).